conf/variant/common/local.conf

```diff
@@ -25,7 +25,8 @@ POKYQEMUDEPS_remove = "packagegroup-core-device-devel"
 NOHDD = "1"
 NOISO = "1"
 
-# Removing tar.bz2 image format as it is not used. Incase of need, it can be supplied through own local.conf
+# Removing tar.bz2 image format as it is not used.
+# If of needed it can be supplied through own local.conf
```

Comment:
Grammar got weird, maybe remove the "of".

Reply:
Yes. I will remove the "of".